src/trade/bid/bid.resolver.ts

```diff
@@ -60,11 +60,21 @@ export class BidResolver {
 
         const { bitPrice, auctionId } = input
 
+        const auction = await this.bidsService.findAuction(auctionId)
+        if (auction.statusId === 3 || auction.statusId === 4) {
+            throw new Error('Auction already closed')
+        }
+
         const participants = await this.bidsService.countParticipantsWithoutUser(auctionId, userId)
         if (participants >= this.config.get<number>('MAX_PARTICIPANTS')) {
             throw new Error('cannot create a bid: max number of participants')
         }
 
+        const maxBid = await this.bidsService.maxBid(auctionId)
+        if (maxBid.bitPrice >= bitPrice) {
+            throw new Error('Your bet must be greater than the maximum bet')
```

Comment:
I think it's better to move error messages to constants (Just const object with properties or enum would be enough)

Reply:
i suggested it before but we decided do it later, because it large work and need do it in the whole project